mne_bids/write.py

```diff
@@ -939,6 +1096,28 @@ def write_raw_bids(raw, bids_basename, output_path, events_data=None,
         bids_raw_folder = bids_fname.split('.')[0]
         bids_fname = op.join(bids_raw_folder, bids_fname)
 
+    # Anonymize
+    if anonymize is not None:
+        if 'daysback' not in anonymize or anonymize['daysback'] is None:
+            raise ValueError('`daysback` argument required to anonymize.')
+        daysback = anonymize['daysback']
+        daysback_min, daysback_max = _get_anonymization_daysback(raw)
+        if daysback < daysback_min:
+            warn('`daysback` is too small; the measurement date ' +
```

Comment:
the + at the end of the line is not necessary as you have (). You should remove them